### ISSUE TITLE
Participants interact with objects

### ISSUE THREAD

**Issue by skleinfeldt** [Dallinger/Griduniverse#234]
User story:

As a participant, if I occupy the same square as any object, I can see information about the object and I can push the action button to interact with the object.

**Comment by alecpm** [comment]
I have a question on PR #253 related to non-"crossable" items. Some of these non-crossable items have available transitions. In such cases, is the transition availabile when adjacent to the square with the item? If so, how do we disambiguate the target when there are multiple adjacent items? For now it might be best to avoid allowing transitions on impassable items, since if a transition requires being on the same square then it could never be executed, and if it doesn't, it could add significant complexity.

**Comment by nataliavelez** [comment]
@alecpm Good point - I agree, I think solely defining transitions on crossable items would be simpler

**Comment by alecpm** [comment]
OK, so in the demo/default config we'll make the Gooseberry Bush crossable, and maybe keep the Empty Gooseberry Bush non-crossable for fun.

**Comment by nataliavelez** [comment]
@jessesnyder Here are the mock-ups that we discussed during the meeting! To recap: The first image shows how transitions might appear to participants when they are (a) visible (b) hidden or (c) visible if previously seen. In the second image, the object in the participant's hand is highlighted to indicate that it's edible.

![transitions_1](https://github.com/Dallinger/Griduniverse/assets/9062288/1eac7d40-5900-4dcb-aba3-22dde12c5bb0)
![transitions_2](https://github.com/Dallinger/Griduniverse/assets/9062288/36a4cda8-8a88-445d-8de1-b1ecb475e0f6)

**Comment by nataliavelez** [comment]
@alecpm Sounds good!
